.travis.yml

```diff
@@ -124,8 +124,6 @@ jobs:
           before_script:
               - npm --version
               - node --version
-          script:
```

Comment:
`npm run ci:build` will fail if any linting issues.

Reply:
:point_up: this is why `npm run ci:lint` has been removed, also another reason is that CRA doesn't have an explicit lint step, it's built-in with dev and prod scripts.